.circleci/cimodel/data/simple/docker_definitions.py

```diff
@@ -26,6 +26,7 @@
     "pytorch-linux-bionic-rocm4.0.1-py3.6",
```

Comment:
Please remove the rocm3.9 and rocm4.0.1 enries from here as we want to only keep the last 3 ROCm versions. You'll need to regenerate config.yml